docs/keeper_dependencies.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
```

Comment:
This is specific to staking. I propose we move them to staking in a specific folder.

Reply:
I think it's not specific to staking. I would propose to put it in the `using-the-sdk` folder, that's where the modules show up in docs.cosmos.network.

Reply:
oh this is simapp. Sorry I misread this.